### ISSUE TITLE
Some instructions, please!

### ISSUE THREAD

**Issue by gusilveiramp** [amjadkhan120/react-native-geofence#2]
Hi,

First of all, congratulations for the work!
This library is ready to use?
Can you give me some instructions about the methods to start with it?

**Comment by amjadkhan120** [comment]
Here is a rough example, how you can use it. Hope it will help you :) 
Regards

`// use to import the module.
import geoFence from 'react-native-geofence';

// call this method to remove the fences register with device.
this.geoFence.removeGeofences();

// call this method to add geo fence, 
/*
 * params structure {
        radius : 100,
        latitude: 123.22,
        longitude: 22.34,
        identifier: unique_identifier,
        notifyOnEntry: true,
        notifyOnExit: true,
        strokeColor:GEOFENCE_STROKE_COLOR,
        fillColor: GEOFENCE_FILL_COLOR
     }
 */
this.geoFence.addGeofence(params, (identifier) => {
            // success callback, add the marker to view.
        }, (error) => {
            console.warn('- addGeofence error: ', error);
        });

// on can be used to add listener to various event.
// Following is the example for adding a listener to location changes.
this.geoFence.on("location", this.methodName);

// to fetch the current location,
// the result will be receive in the location callback listener.
// so first you have to add the location listener in order to get the location.
this.geoFence.requestLocation();


// un can be used to remove any listener added previously.
// again the same example for removing the location listener
this.geoFence.un("location",this.onLocation);`

**Comment by gusilveiramp** [comment]
Thank you very much!

**Comment by gusilveiramp** [comment]
Another doubt, It is possible to use it with Background monitoring?
Geofence's will alert user even if the app is closed (killed, totally closed) or in background?

**Comment by amjadkhan120** [comment]
@gusilveiramp Yes, App will post a local notification Even if the app is closed & hard kill by user. Plus app can post to a specific URL mentioned while configuring the module, this will require the URL, Method(POST,GET), & other necessary header values, which are require to send the request.
